roles/senior_engineer.md

```diff
@@ -1,3 +1,49 @@
 # Senior Engineer
 
-Coming soon...
+Our software delivery teams are the core of our business. They are usually comprised of a Lead Engineer, a Delivery Manager and one or more Engineers.
+
+As an Engineer we expect someone:
+
+## To be able to autonomously deliver working software
+
+We expect our Engineers to be able to work on entire features, from conception through to production deployment, without guidance and direction from others. This doesn't mean they have to be delivered without pairing, or asking advice from the wider team. What it does mean is the primary force for ensuring the feature is built and deployed in a timely fashion is the responsiblity of the Engineer working on the problem.
+
+## To be fully billable
+
+We expect our Engineers to be fully billed on customers deliveries as we are a business trying to make a profit by delivery valuable working software to our customers.
+
+## To be involved in practice improvement discussions
+
+We expect everyone in our team to reflect on our practices and be a proactive voice in suggesting change. We expect you to be reflecting on your own development practices, reflecting on your team and the wider business.
+
+## To be actively practicing TDD
+
+We expect our engineering team to write tests first by default. We expect every feature to be tested. Every Pull Request (or equivalent chunk of work put up for peer review) must contain a test. If for some reason there are extenuating circumstances for not writing tests then more than one engineer should peer review and verify this.
```

Comment:
I know you said you're not looking for comments, but want to nitpick. Are we expecting `test first` or `TDD`? It's important be clear about the difference. 

TDD is the discipline that we've been focussing on in Learn Tech and test first is not a discipline.